spec.bs

```diff
@@ -1024,73 +1114,185 @@ Issue: Once <a href="https://github.com/wicg/turtledove/issues/615">
     {{InterestGroupScriptRunnerGlobalScope}}s until the script's initial
     execution is complete.
 
-<div algorithm>
-The <dfn method for="PrivateAggregation">contributeToHistogramOnEvent(DOMString
-event, PAExtendedHistogramContribution contribution)</dfn> method steps are:
-</div>
-
-Issue: Fill in validation. Will need to check offsets are valid and in the right
-    range for the type etc. Also that base values are one of a set.
+Protected Audience API-specific structures {#protected-audience-api-specific-structures}
+----------------------------------------------------------------------------------------
 
-Issue: Need to document limits on offset, etc.
+<h4 id="extending-auction-config">Extending auction config</h4>
 
-Issue: Fill in the rest. (Need to put the contribution in some sort of queue and
-    process the queue at some point. Need to decide where the queue should live
-    given it has to outlive the auction.)
+Extend the <a spec="turtledove">auction config</a> [=struct=] to add a new
```

Comment:
Is this implemented. If so, did we save those to auction config?
I think auction config is not something we'd like to modify.
If it's something provided by the browser, how about [browserSignals](https://github.com/WICG/turtledove/blob/main/spec.bs#L1011)?

Reply:
I think browserSignals is something that gets passed as a JavaScript argument to bidding script.

I went with extending the auction config as it's currently used to store some per-auction state (e.g. the pending promise count https://wicg.github.io/turtledove/#auction-config-pending-promise-count). I don't think the spec's auction config is exactly 1:1 with the implementation's AuctionConfig (the implementation's doesn't include this state).

I also couldn't really see any other spec concept that stores per-auction state.

Reply:
oh I messed up the AuctionConfig IDL and the struct somehow. The struct [=auction config=] seems fine.